flake.nix

```diff
@@ -622,6 +682,7 @@
         };
       in
       {
+        nixConfig.substituters = [ "https://fedimint.cachix.org/" ];
```

Comment:
Putting in :tinfoilhat: . Not sure if we should do it here. It puts an extra trust in that host.

Bootstrap script already prompts users to do it: https://github.com/fedimint/fedimint/blob/0cf81c52305af181f787b07e704d2d5af35083eb/scripts/bootstrap.sh#L30 and then it's more of their own choice.

Reply:
I kind of wonder if this even works. Do you have a link to where did you find it? I guess we'll know after it hits master.

Reply:
Dropped the commit introducing it.